server/hedley/modules/custom/hedley_municipality/hedley_municipality.module

```diff
@@ -306,7 +306,7 @@ function hedley_municipality_node_insert($node) {
 
   // Run the code to create all the vocabularies and link it with the content
   // types.
-  $vocab_name = 'Topics for ' . strip_tags($node->title);
+  $vocab_name = t('Topics for') . ' ' . strip_tags($node->title);
```

Comment:
why translatable?

Reply:
(It will still create the vocab name with one language)

Reply:
@noamoss ^^ this won't work, the editors will still see it as "Topics for ...", do want to change the language?

Reply:
We need Hebrew for sure. If there is a way to show the string "topics for [muni]" also in English and Arabic it is meaning full for the editors.

Reply:
@nedSaf let's just to רשימת נושאים [שם הרשות] - all in Hebrew.

Reply:
Ok but we can't guarantee that `[muni name]` will be in Hebrew, it will be in whatever language the Muni was created.

Reply:
I have an idea, I'll check if it will work first.